src/js/structs/__tests__/ServiceTree-test.js

```diff
@@ -147,26 +186,135 @@ describe('ServiceTree', function () {
     });
   });
 
+  describe('#filterItemsByFilter', function () {
+
+    beforeEach(function () {
+      this.instance = new ServiceTree({
```

Comment:
this code is repeated in almost all `describe` blocks. let's just put this in a `beforeEach` in the top-most `describe` block